src/blocks/StickyHighlights/Highlight/index.tsx

```diff
@@ -100,19 +99,23 @@ export const StickyHighlight: React.FC<Props> = React.memo(
           classes[`scroll-direction--${init ? yDirection : 'down'}`],
         ].join(' ')}
       >
-        <Grid className={classes.minHeight}>
-          <Cell cols={5} colsM={8}>
+        <div className={[classes.minHeight, 'grid'].filter(Boolean).join(' ')}>
+          <div className={'cols-7 cols-m-12'}>
             <RichText content={richText} className={classes.richText} />
             {enableLink && <CMSLink {...link} appearance="default" mobileFullWidth />}
-          </Cell>
-        </Grid>
+          </div>
+        </div>
         <CSSTransition in={visible} timeout={750} classNames="animate">
           <Gutter className={classes.codeMediaPosition}>
             {type === 'code' && (
               <React.Fragment>
                 <PixelBackground className={classes.pixels} />
-                <Grid>
-                  <Cell cols={6} start={7} colsM={8} startM={1} className={classes.bg}>
+                <div className={['grid'].filter(Boolean).join(' ')}>
+                  <div
+                    className={[classes.bg, 'cols-8 start-9 cols-m-8 start-m-8']
```

Comment:
@paulpopus Why are you setting the `start-m` value to `8` here instead of `1` like it was before?